app/components/Search.jsx

```diff
@@ -90,7 +90,7 @@ class Search extends Component {
     const partyId = currentParty.id;
     const { uid } = user;
     const { dj_name } = this.props.djs[uid];
-    const song = { song_uri, title, artist, artwork_url, uid, dj_name, time_priority: 0, vote_priority: 0};
```

Comment:
I think we need to keep uid on the song object, otherwise the shadow queue --> personal queue transition breaks